tests/test_models.py

```diff
@@ -0,0 +1,20 @@
+from lmsweb.models import Course, User
+
+
+class TestCourse:
+    def test_add_user(self, user: User, course: Course):
+        course.add_user(user)
+        assert user.course == course  # NOQA: S101
```

Comment:
Maybe we should configure it in a file level configuration for flake8

Reply:
Done